docs/index.md

```diff
@@ -1128,6 +1128,50 @@ app/components
 </div>
 ```
 
+##### Translations (experimental)
+
+To use experimental support for `I18n` translations, include `ViewComponent::Translatable`:
+
+`app/components/example_component.rb`
+
+```ruby
+module ExampleComponent < ApplicationComponent
+  include ViewComponent::Translatable
+end
+```
+
+Add a sidecar YAML file with translations for the component:
+
+`app/components/example_component.yml`
+
+```yml
+en:
+  hello: "Hello world!"
+```
+
+Access translations as usual:
+
+`app/components/example_component.html.erb`
+
+```erb
+<p><%= t("hello") %></p>
```

Comment:
Is this still accurate? Do we need to update the docs to use the "lazy" format? e.g. `t(".hello")`?